views/base.njk

```diff
@@ -53,4 +77,5 @@
             {% endfor %}
         {% endif %}
     {% endblock %}
+    
```

Comment:
Wait, there shouldn't be stuff outside the body tag?!

Reply:
yeah, not sure why that stuff is out there.... will move in.